Sources/StreamChatUI/Utils/DateFormatter+Extensions.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
+    static func makeDefault() -> DateFormatter {
+        let formatter = DateFormatter()
+        formatter.dateStyle = .none
+        formatter.timeStyle = .short
+        formatter.locale = Locale.autoupdatingCurrent
+        return formatter
+    }
```

Comment:
nit: I think this should rather be `static let default: DateFormatter`. We only need one instance of the default formatter.